src/operator/tensor/elemwise_unary_op_basic.cc

```diff
@@ -1069,13 +1069,72 @@ The storage type of ``log2`` output is always dense
 .set_attr<nnvm::FGradient>("FGradient", ElemwiseGradUseIn{"_backward_log2"});
 
 MXNET_OPERATOR_REGISTER_BINARY_WITH_SPARSE_CPU_DR(_backward_log,
-                                                  unary_bwd<mshadow_op::log_grad>);
+                                                  unary_bwd<mshadow_op::log_grad>)
+.set_attr<nnvm::FGradient>("FGradient",
+  [](const nnvm::NodePtr& n, const std::vector<nnvm::NodeEntry>& ograds) {
+    // For g(x) -> g = log
+    // g''(x) = -1 * (g'(x) * g'(x))
+    auto gx = nnvm::NodeEntry{n, 0, 0};
```

Comment:
You are welcome to simplify calls to NodeEntry as per:

https://github.com/apache/incubator-mxnet/pull/14095

Just call nnvm::NodeEntry{n}

Reply:
Oh. Missed that. Thank You.